lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -61,7 +89,14 @@ public static String convertGpaToLetterGrade(double gpa) {
    * @return The factorial of n.
    */
   public static int computeFactorial(int n) {
-    return 0;
+    if (n < 0) {
+      return -1;
```

Comment:
This is the wrong behavior for n < 0.

Reply:
deleted logic

Reply:
Actually, the proper behavior is to throw an exception. I'll allow this for now.